client/src/context/Reservation/index.js

```diff
@@ -0,0 +1,15 @@
+import { createContext, useState } from 'react';
+
+export const ReservationContext = createContext();
+const ReservationProvider = ({ children }) => {
```

Comment:
I think it should also hold the other info like screeningId, cinemaHallId and movieId.